### ISSUE TITLE
Distributed Resample Query Bug

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1122]
When running VisIt in parallel, if I perform a resample with distributed resample enabled and then query the number of nodes or zones of the actual data, the result of the query is very incorrect. 

Example:
Open up rect2d.silo and add a Pseudocolor plot. 
Add a 2D resample operator of 10x10 samples, and disable 'distributed resample'.
Draw the plot and query the number of zones. You should get 100.
Now resample with 'distributed resample' enabled. 
Again query the number of zones. This returns 20 for me.  

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Alister Maguire
Original creation: 03/15/2018 02:10 pm
Original update: 03/20/2018 06:34 pm
Ticket number: 3045
